### DISCUSSION TITLE
New Update not working on San Franciscan Roasters

### DISCUSSION THREAD

**Discussion by pkennnn** [artisan-roaster-scope/artisan#1726] [General]
Hello,
I wanted to bring attention to the fact that the newest update is not reading the probes on San Franciscan roasters and I would love this to be addressed or have an older artisan download available to San Franciscan Roaster owners.

Thank you!

**Comment by bigl00z3** [comment]
Can you please fill a [bug report/issue](https://github.com/artisan-roaster-scope/artisan/issues) with the details of your setup, the exact version that you are using (as detailed in the issue template) in order for contributors to evaluate and understand the problem you are encountering on your roaster?

Also don't forget to upload your .aset artisan settings as explained in the issue template using a .txt extension

Nico

**Reply by MAKOMO** [reply]
Thanks Nick, but in this case raising an issue is not needed as we are aware of this hickup with some roasters communicating over (serial) MODBUS in our latest release.

**Comment by MAKOMO** [comment]
Dear pkennnn,

we are aware that Artisan v3.0.2 has issues to communicate with some machines over MODBUS due to a bug in the underlying MODBUS lib we upgraded to. This has been fixed. The fix will be available in the next release which we hope to push out soon.

For now you can upgrade to the current [Continuous Build](https://github.com/artisan-roaster-scope/artisan/releases/tag/continuous) which fixes this and some other minor things.

**Reply by pkennnn** [reply]
Thank you I really appreciate it!
